backend/src/api/users.ts

```diff
@@ -141,9 +142,17 @@ export const invite = wrapHandler(async (event) => {
     `
       )
       .execute();
-  }
 
-  // TODO: Send invite email via SES
+    await sendEmail(
+      user.email,
+      'Crossfeed Invitation',
+      `Hi there,\n\nYou've been invite to join the ${organization?.name} organization` +
+        ` on Crossfeed. To accept the invitation and start using Crossfeed, sign on at ` +
+        `https://${
+          process.env.NODE_ENV !== 'production' ? 'staging.' : ''
```

Comment:
Let's also use an environment variable called `FRONTEND_DOMAIN` -- this way, we can use it elsewhere in the code as needed.